workflows/illuminaNcov.nf

```diff
@@ -118,11 +118,11 @@ workflow sequenceAnalysis {
 
       writeQCSummaryCSV(qc.header.concat(qc.pass).concat(qc.fail).toList())
 
-      mappingStatistics(trimPrimerSequences.out.ptrim.combine(ch_preparedRef.map{ it[0] }))
-
       pangolinTyping(makeConsensus.out.consensus_fasta)
+      
+      nextclade(makeConsensus.out.consensus_fasta)
 
-      multiqc(readTrimming.out.log.collect(), mappingStatistics.out.collect(), fastqc.out.collect())
+      multiqc(readTrimming.out.log.collect(), readMapping.out.log.collect(), fastqc.out.collect())
```

Comment:
I see you have removed mappingStatistics.out.collect() . Does this remove any results from MultiQC?

Reply:
I found this here: https://github.com/genomic-medicine-sweden/gms-artic/commit/d7e2233a07d71b2211e0cd3a53f0f362f3a815ca

Reply:
Why are we dropping Picard from MultiQC?

Reply:
I think it's from #12 add_mem_params. I didn't have any issue with Picard